content/reference/endpoints/payment_gateways.md

```diff
@@ -3,32 +3,58 @@
 1. TOC
 {:toc}
 
+## Overview
+
+This resource represents Payment Gateways (allowing integration with providers such BookingPay, Stripe, Ingenico etc.) that you can later assign to [Rentals](/reference/endpoints/rentals/) and start accepting payments via credit card.
```

Comment:
BookingPay isn't publicly available to all users yet. I think we might want to hold on mentioning it explicitly in the docs before it's accessible for everyone.